contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -55,6 +55,57 @@ library ValidationLogic {
      */
     uint256 public constant HEALTH_FACTOR_LIQUIDATION_THRESHOLD = 1e18;
 
+    function validateStatusForBlurExchangeRequest(
+        bool isEnable,
+        address keeper,
+        uint256 ongoingRequestAmount,
+        uint256 ongoingRequestLimit
+    ) internal pure {
+        require(isEnable, Errors.BLUR_EXCHANGE_REQUEST_DISABLED);
+        require(keeper != address(0), Errors.INVALID_KEEPER_ADDRESS);
+        require(
+            ongoingRequestAmount <= ongoingRequestLimit,
+            Errors.ONGOING_REQUEST_AMOUNT_EXCEEDED
+        );
+    }
+
+    function validateInitiateBlurExchangeRequest(
```

Comment:
suggest to check NFT type isn‘t stakeFish when  init buy-require
user can withdraw 32ETH before keeper buy it